### DISCUSSION TITLE
Adding blocks within GJS div

### DISCUSSION THREAD

**Discussion by MilosVujnovicTC** [GrapesJS/grapesjs#3591] [Q&A]
Hi,

So when I insert blocks into the editor, it seems that all blocks get inserted after the main "gjs" div.
What I'm trying to figure out is how to get blocks to be inserted by default inside of several "wrap" divs like so: 

`    <div id="gjs" style="height:0px; overflow:hidden;">
      <div class="wrap container-fluid">
        <div class="content row">
          <main class="main">
            <div class="container-fluid">
              
            </div>
          </main>
        </div>
      </div>
    </div>`

Looking forward to your suggestions...thanks!

**Comment by artf** [comment]
You can configure each block as you wish, with your HTML of choice, please refer to this guide: https://grapesjs.com/docs/modules/Blocks.html
